### ISSUE TITLE
Does not work with Remote Development (VS Code Insiders)

### ISSUE THREAD

**Issue by tm1000** [PsykoSoldi3r/vscode-itunes#17]
**Description**

When using the preview build of VS Code (https://code.visualstudio.com/docs/remote/ssh) which enables the ability to use SSH and connect to remote machines for code editing. This extension won't enable itself unless it's "installed remotely" and past that point it won't work (because its installed remotely)

I havent looked over the code yet but the VS Code documents talk about how to fix this from an extension state if its installing in the wrong location: https://code.visualstudio.com/api/advanced-topics/remote-extensions

**macOS version**

v10.14.5 (18F132)

**vscode Version**

v1.35.0-insider

**Comment by tm1000** [comment]
The workaround for this is to add this to your settings.json (See: https://code.visualstudio.com/docs/remote/ssh#_advanced-forcing-an-extension-to-run-locally-remotely)

```
"remote.extensionKind": {
    "psykosoldi3r.itunes-vscode": "ui"
}
```

**Comment by joeykamsteeg** [comment]
Fixed and will be available in version 0.10.0
